packages/core/src/data-import/providers/import-parser/import-parser.spec.ts

```diff
@@ -51,23 +77,33 @@ describe('ImportParser', () => {
             expect(result.results).toMatchSnapshot();
         });
 
+        it('works with multilingual input', async () => {
+            const importParser = new ImportParser(mockConfigService);
+
+            const filename = path.join(__dirname, 'test-fixtures', 'multiple-languages.csv');
+            const input = fs.createReadStream(filename);
+            const result = await importParser.parseProducts(input);
+
+            expect(result.results).toMatchSnapshot();
```

Comment:
I wish I never used snapshot tests now. They are just so opaque and I can't just see by glancing at the e2e test what particular outcome is being tested for. I understand why you used it here - just following my lead. You don't need to change it, but you can if you want. Mostly I just felt the need to vent about snapshot tests :D

Reply:
Haha, so true. I found it tedious to check if the snapshot actually contained the intended output. But they do work now, so I feel a bit too lazy to change them now ;-)